.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 npm-debug.log
-build
```

Comment:
no, before npm publish or run it in local dir before npm install in core so contract get copied with the built json then contracts index expose the .jsons